packages/react-components/theme-designer/src/components/Nav/Nav.tsx

```diff
@@ -1,20 +1,71 @@
 import * as React from 'react';
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { CircleRegular, ChevronRightRegular } from '@fluentui/react-icons';
+import { Text, Menu, MenuTrigger, MenuButton, MenuPopover, MenuList, MenuItem } from '@fluentui/react-components';
 
 export interface NavProps {
   className?: string;
 }
 
 const useStyles = makeStyles({
   root: {
-    display: 'flex',
     alignItems: 'center',
-    justifyContent: 'left',
+    backgroundColor: 'black',
+    color: 'white',
+    display: 'grid',
+    gridTemplateColumns: '100px 200px auto 300px',
     ...shorthands.borderBottom('1px', 'solid', '#D1D1D1'),
   },
+  logo: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'space-evenly',
+  },
+  content: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  export: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'flex-end',
+    paddingRight: '20px',
+  },
+  exportButton: {
+    color: 'white',
+  },
 });
 
 export const Nav: React.FC<NavProps> = props => {
   const styles = useStyles();
-  return <div className={mergeClasses(styles.root, props.className)}>Nav</div>;
+  return (
+    <div className={mergeClasses(styles.root, props.className)}>
+      <div className={styles.logo}>
+        <CircleRegular primaryFill="white" />
```

Comment:
You should not need to set `primaryFill` here. All the SVGIcons default to using [`currentColor`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword) for fill. In this case the `color` property resolves to white so there's no need to override the icon color.

<img width="215" alt="svg_current_color" src="https://user-images.githubusercontent.com/93940821/171501627-d362a408-c2db-4ec0-9ae9-4d9e5038fee7.png">

If you did need to change the color it would be better to set it on the parent in this case so both the icon and `Text` will use the same color.